src/VisualStudio.Roslyn.SDK/SyntaxVisualizer/Roslyn.SyntaxVisualizer.Extension/SyntaxVisualizerContainer.xaml.cs

```diff
@@ -62,7 +64,7 @@ internal SyntaxVisualizerContainer(SyntaxVisualizerToolWindow parent)
 
             UpdateThemedColors();
 
-            syntaxVisualizer.SyntaxNodeNavigationToSourceRequested += node => NavigateToSource(node.Span);
+            syntaxVisualizer.SyntaxNodeNavigationToSourceRequested += node => NavigateToSource(node?.Span);
```

Comment:
❔ Can this really be null?

Reply:
yep there are a lot of places where we call roslyn apis that may return null then ask to navigate to them